dependaban/bin/can-i-ignore-scripts.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Print output and save to variable for inspection. We need to inspect because
+# it always exits with zero.
+# https://github.com/naugtur/can-i-ignore-scripts/pull/25
+CHECK_OUTPUT="$(npx can-i-ignore-scripts | tee /dev/stderr)"
```

Comment:
That issue has been fixed, I left a comment on the PR:
https://github.com/naugtur/can-i-ignore-scripts/pull/17#issuecomment-1446490554

Reply:
Oh! I actually mean that it's looking too deeply into a package rather than stopping at `node_modules/<package-name>/package.json` or `node_modules/<package-name>/node_modules/<package-name-2>/..further-nesting.../node_modules/<package-name-n>/package.json`.

Like, some packages are packaged incorrectly and would possibly have `<package-name>/example/do-this/package.json`, and this script would accidentally take that.

Here's a few examples from `goop` itself. Most of them might be okay if they're dev dependencies (but what if the scripts adjust the source code during a compile step?), but the `@kubernetes/client-node` dependency might not be too safe.

![image](https://user-images.githubusercontent.com/26241442/221766589-1689067a-4927-4ded-891e-ccc670a7bd0f.png)

Reply:
Gotcha. I had planned on adding an "exceptions" input var to allow overriding, but haven't needed it yet. If the upstream repo is unmaintained and we want to fix stuff like this, we can fork or warehouse it in this action.